CI/before_script.msvc.sh

```diff
@@ -761,3 +761,4 @@ if [ -z $VERBOSE ]; then
 	fi
 fi
 exit $RET
+
```

Comment:
Unnecessary line break

Reply:
@AnyOldName3 I obviously copy past your work :)
I'll redo this pr cleaner after bugs solved